### DISCUSSION TITLE
IEP 1 - IEP Purpose and Guidelines

### DISCUSSION THREAD

**Discussion by MatYarger** [DigitizingForImpact/IEPs#3] [Process IEP]
The initial draft of IEP 1, outlining the IEP Purpose and Guidelines is currently being drafted in google docs as part of the Open Collaboration on Digitizing Impact and reviewed by members of the Digital Infrastructure and Open APIs (DIAPI) Working Group. This draft gdoc can be found [here](https://docs.google.com/document/d/1SUAhRnQF0eRsjXPXt7FzX2I4dcxLGndPdJyzAqG55lw/edit?usp=sharing) for review.

**Comment by 0xAurelius** [comment]
Left some feedback and copy-edits on the doc back during the initial consultation - but broadly this seems like a sound and reasonable framework for the IEP workflow.
